### DISCUSSION TITLE
Running omnetpp error gate already connect

### DISCUSSION THREAD

**Discussion by dion524** [omnetpp/omnetpp#1252] [Q&A]
![image](https://github.com/user-attachments/assets/e7425a80-e949-46bf-adc8-4af648f02c77)

Hello everyone.
I am trying to simulate the Mqtt protocol in order to measure its latency. The codes of the different files are compiled without error. After execution, the environment is displayed but an error is displayed as in the screenshot. Can someone help me?

**Comment by rhornig** [comment]
This is not a bugreport against OMNeT++ so I'm converting it to a discussion topic

**Comment by rhornig** [comment]
Based on the information you provided, there is an error in your code in `MqttNetwork.ned` in line 29.

Jokes aside, I think if you expect someone to answer this question, at a minimum you should provide the content of the file which generated the error.
